resources/zib-AddressInformation.xml

```diff
@@ -0,0 +1,339 @@
+               <path value="url"/>
+            </discriminator>
+            <rules value="open"/>
+         </slicing>
+      </element>
+      <element id="Address.line.extension:streetName">
+         <path value="Address.line.extension"/>
+         <sliceName value="streetName"/>
+         <short value="Street"/>
+         <definition value="Street name of the address."/>
+         <alias value="Straat"/>
+         <max value="1"/>
+         <type>
+            <code value="Extension"/>
+            <profile value="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-streetName"/>
+         </type>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.2"/>
+            <comment value="Street"/>
+         </mapping>
+      </element>
+      <element id="Address.line.extension:houseNumber">
+         <path value="Address.line.extension"/>
+         <sliceName value="houseNumber"/>
+         <short value="HouseNumber"/>
+         <definition value="House number of the address."/>
+         <alias value="Huisnummer"/>
+         <max value="1"/>
+         <type>
+            <code value="Extension"/>
+            <profile value="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-houseNumber"/>
+         </type>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.12"/>
+            <comment value="HouseNumber"/>
+         </mapping>
+      </element>
+      <element id="Address.line.extension:houseNumberLetter-houseNumberAddition">
+         <path value="Address.line.extension"/>
+         <sliceName value="houseNumberLetter-houseNumberAddition"/>
+         <short value="HouseNumberLetter-HouseNumberAddition"/>
+         <definition value="Export&#xD;&#xA;&#xD;&#xA;- If a HouseNumberLetter as well as a HouseNumberAddition is known: HouseNumberLetter first, followed by a space and finally the HouseNumberAddition.&#xD;&#xA;- If only a HouseNumberLetter is known, send just that. No trailing space is required.&#xD;&#xA;- If only a HouseNumberAddition is known, communicate that with a leading space.&#xD;&#xA;&#xD;&#xA;Import&#xD;&#xA;- If the value of buildingNumberSuffix contains no space, then the whole value is considered to be the HouseNumberLetter.&#xD;&#xA;- If the value of buildingNumberSuffix contains a space, then everything before the first space is considered to be the HouseNumberLetter, and the everything trailing the first space is considered to be the HouseNumberAddition.&#xD;&#xA;&#xD;&#xA;On the leading space&#xD;&#xA;While a leading space is allowed in string values in FHIR, the parsing of such space cannot be guaranteed in XML representations. Although aware of this issue, this mapping is not changed because of compatibility reasons. If an implementer cannot parse a leading space, the practical risk of mapping a HouseNumberIndication to a HouseNumberAddition is deemed insignificant."/>
```

Comment:
buildingNumberSuffix == HouseNumberAddition?

Reply:
Tweemaal veranderd in 'the houseNumberLetter-houseNumberAddition extension' (gezien de sliceName gewijzigd is naar de functionele naam, maar alleen die sliceName neerzetten werkt misschien weer iets verwarrend).